src/main/java/frc/robot/FieldConstants.java

```diff
@@ -51,6 +51,9 @@ public static class Barge {
     // Measured from floor to bottom of cage
     public static final double deepHeight = Units.inchesToMeters(3.125);
     public static final double shallowHeight = Units.inchesToMeters(30.125);
+
+    public static final Pose2d alignmentX =
+        new Pose2d(new Translation2d(7.743555545806885, -1), new Rotation2d());
```

Comment:
[nit] This value is specific to our robot, right?  It doesn't really matter this year, but if we were maintaining this for other people, then my suggestion would be to use the middle of the field  + some standard offset for the alignment X value.